checker/src/org/checkerframework/checker/nullness/NullnessAnnotatedTypeFactory.java

```diff
@@ -154,6 +154,13 @@ public NullnessAnnotatedTypeFactory(BaseTypeChecker checker, boolean useFbc) {
                 org.checkerframework.checker.nullness.compatqual.MonotonicNonNullType.class,
                 MONOTONIC_NONNULL);
 
+        // Alias deprecated annotation
```

Comment:
That's fine with me.  Or you could make AnnotatedTypeFactory#getDeclAnnotation(javax.lang.model.element.Element, java.lang.String, boolean) public.  I don't have a preference.